cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLogger.java

```diff
@@ -0,0 +1,44 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import okhttp3.logging.HttpLoggingInterceptor;
+
+final class CwmsHttpLogger implements HttpLoggingInterceptor.Logger {
```

Comment:
This is awfully specific, did you consider making this more extensible? i.e. parameterize in the keys you want to redact like you did for the headers?

Reply:
Will do if we add a 3rd